doc/source/ray-air/getting-started.rst

```diff
@@ -3,147 +3,71 @@ Ray AI Runtime (alpha)
 
 Ray AI Runtime (AIR) is an open-source toolkit for building end-to-end ML applications. By leveraging Ray and its library ecosystem, it brings scalability and programmability to ML platforms.
 
+.. image:: /images/ai_runtime.jpg
+
 
 The main focuses of the Ray AI Runtime:
 
 * Ray AIR focuses on providing the compute layer for ML workloads.
 * It is designed to interoperate with other systems for storage and metadata needs.
 
 
-Ray AIR consists of 5 key components -- Data processing (Ray Data), Model Training (Ray Train), Reinforcement Learning (Ray RLlib), Hyperparameter Tuning (Ray Tune), and Model Serving (Ray Serve).
+Ray AIR consists of 5 key components -- Data processing (:ref:`Ray Data <datasets>`), Model Training (:ref:`Ray Train <train-docs>`), Reinforcement Learning (:ref:`Ray RLlib <rllib-index>`), Hyperparameter Tuning (:ref:`Ray Tune <tune-main>`), and Model Serving (:ref:`Ray Serve <rayserve>`).
 
 Users can use these libraries interchangeably to scale different parts of standard ML workflows.
 
 
 .. tip::
     **Getting involved with Ray AIR.** We'll be holding office hours, development sprints, and other activities as we get closer to the Ray AIR Beta/GA release. Want to join us? Fill out `this short form <https://forms.gle/wCCdbaQDtgErYycT6>`__!
 
+Quick Start
```

Comment:
Love the quick start ❤️

Reply:
I love that we have a quickstart but I don't love the examples, they don't seem very "quick". But that's our of scope for this PR, let's check in the documentation structure first and iterate on the examples later